snap/snapcraft.yaml

```diff
@@ -336,10 +335,15 @@ apps:
     plugs: [network, network-bind]
   app-service-configurable:
     adapter: full
-    command: bin/app-service-configurable -confdir $SNAP_DATA/config/app-service-configurable/res -profile rules-engine --registry
+    command: >-
+      bin/app-service-configurable
+      -confdir $SNAP_DATA/config/app-service-configurable/res
+      -profile rules-engine
+      -cp http://localhost:8500
```

Comment:
Still need `--registry`  for service registration/discovery

Reply:
Hmmm, I thought you said if I added -cp, I wouldn't need to specify --registry???

Reply:
No, we separated them config provider is separate from registry provider.